server/loader.py

```diff
@@ -168,8 +55,27 @@ def validate_settings(event):
 
 
 def load(info):
+    # Load the mako template for Minerva and serve it as the root document.
+    minerva_mako = os.path.join(os.path.dirname(__file__), "minerva.mako")
+    minerva_webroot = Webroot(minerva_mako)
+
+    # Load resources from all enabled plugins, which allows Minerva
+    # to be extended by other Girder plugins.
+    plugins = ModelImporter.model('setting').get(SettingKey.PLUGINS_ENABLED,
+                                                 ())
+    plugins = list(plugins)
+    if 'minerva' in plugins:
+        plugins.remove('minerva')
+    vars = {
+        'plugins': plugins,
+        'apiRoot': '/api/v1',
+        'staticRoot': '/static',
+        'title': 'Minerva'
+    }
+    minerva_webroot.updateHtmlVars(vars)
```

Comment:
A couple of issues here.

1. This is called [here](https://github.com/girder/girder/blob/master/girder/utility/plugin_utilities.py#L79) after girder core has determined what plugins are enabled, so none of the changes made here make a difference at least for the purpose of plugin loading.
2. Hard coding the api and static root like this will break a lot of deployment scenarios.

I would suggest simplifying this part to do something like the following:
```python
minerva_mako = os.path.join(os.path.dirname(__file__), "minerva.mako")
minerva_webroot = Webroot(minerva_mako)
minerva_webroot.updateHtmlVars(info['serverRoot'].vars)
minerva_webroot.updateHtmlVars({'title': 'Minerva'})
```